properties-files/openshiftv4_scale.properties

```diff
@@ -0,0 +1,24 @@
+CLUSTER_USER=kubeadmin
+CLUSTER_PASSWORD=
+CLUSTER_API_URL=
+SSHKEY_TOKEN=
+ORCHESTRATION_HOST=localhost
+ORCHESTRATION_USER=root
+WORKLOAD_IMAGE=quay.io/openshift-scale/scale-ci-workload
+WORKLOAD_JOB_NODE_SELECTOR=false
+WORKLOAD_JOB_TAINT=false
```

Comment:
`WORKLOAD_JOB_NODE_SELECTOR` and `WORKLOAD_JOB_TAINT` are both defaulted to `true` because technically you can run this workload to scale down a cluster and if we do not isolate the pod to the workload node, the nodes sclae down (removed) could be the node hosting the workload pod.